yb-voyager/src/query/queryissue/issues_dml.go

```diff
@@ -304,3 +304,17 @@ var nonDecimalIntegerLiteralIssue = issue.Issue{
 func NewNonDecimalIntegerLiteralIssue(objectType string, objectName string, sqlStatement string) QueryIssue {
 	return newQueryIssue(nonDecimalIntegerLiteralIssue, objectType, objectName, sqlStatement, map[string]interface{}{})
 }
+
+var twoPhaseCommitIssue = issue.Issue{
+	Type:        TWO_PHASE_COMMIT,
+	Name:        "Two-Phase Commit (XA syntax)",
+	Impact:      constants.IMPACT_LEVEL_2, //TODO: confirm
```

Comment:
I think this could be 3. Doesn't seem like there is a workaround? Let's discuss this in standup.